src/renderer.ts

```diff
@@ -350,7 +350,7 @@ class SparklineCellRenderer extends DefaultCellRenderer {
 //console.log(i,x(i));
         var line = d3.svg.line<number>()
```

Comment:
why are you creating a new d3.line instance for each call? could be extracted to a single one